change/react-native-windows-2020-05-12-14-18-36-OC-4094136-connectxcpt.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
FYI, `none` means Beachball won't publish a build for this or include it in its changelog. This should be `prerelease`.

Reply:
I was on the fence on this one.
When this change is backported to `0.60-stable`, I do plan to generate a new version.
for the master branch, didn't think it was worth it.

What criteria would you suggest?
Any change we want reflected on any branch as a new package version should be prerelease?

Reply:
Basically the only time we want to use “none” is when the change doesn’t impact the package (e.g. changing CI). We switched the master branch to a nightly build to reduce version spam, so you shouldn’t need to worry about this making a new build. Having the commit in a change log is nice though, and we would probably want to publish this if it was the only thing committed that day.

Reply:
Basically the only time we want to use “none” is when the change doesn’t impact the package (e.g. changing CI). We switched the master branch to a nightly build to reduce version spam, so you shouldn’t need to worry about this making a new build. Having the commit in a change log is nice though, and we would probably want to publish this if it was the only thing committed that day.